spec/takeaway_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  let(:menu_klass) { double :menu_klass, new: menu }
+  let(:menu) { double :menu, show: "{\"chilli\"=>2.99, \"pizza\"=>4.5}"}
+
+  subject { described_class.new(menu_klass)}
+
+  before(:each) do
+    subject.order("chilli", 3)
+    subject.order("pizza")
+    allow(menu).to receive(:price).with('chilli').and_return 2.99
+    allow(menu).to receive(:price).with('pizza').and_return 4.5    
+  end
+
+  describe 'ordering' do
+
+    it 'order should display confirmation message' do
+      expect{ subject.order("chilli", 3) }.to output("3x chilli(s) added to your basket.\n").to_stdout
+    end
+
+    it 'order should add to basket' do
+      expect(subject.basket).to include("chilli"=>3)
+    end
+
+    it 'should not be able to order item not on menu' do
+      expect{ subject.order('lasagne') }.to raise_error "Sorry, we don't have that!"
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)